api/envoy/extensions/filters/network/dubbo_proxy/v3/route.proto

```diff
@@ -38,6 +38,9 @@ message RouteConfiguration {
   // The list of routes that will be matched, in order, against incoming requests. The first route
   // that matches will be used.
   repeated Route routes = 5;
+
+  // The Matcher of interface.
```

Comment:
Can we expand this a bit? What is "Matcher", what the purpose of it etc. Or at very least put Matcher in lowercase?